In [2]:
import pandas as pd
import numpy as np
from scipy import stats
from scipy.stats import norm
from scipy.stats import chi2_contingency

# 2. Import Data 

In [3]:
Data=pd.read_csv('BuyerRatio.csv')
Data

Observed Values  East  West  North  South
0           Males    50   142    131     70
1         Females   435  1523   1356    750

In [4]:
Data.head()

Observed Values  East  West  North  South
0           Males    50   142    131     70
1         Females   435  1523   1356    750

In [6]:
Data.shape

(2, 5)

## Hypothesis: H0: Male-female buyer rations are similar across regions. 
## H1: Male-female buyer rations are not similar across regions.

In [7]:
#we will make a Dimensional Array
Observations=np.array([[50,142,131,70],[435,1523,1356,750]])
Observations

array([[  50,  142,  131,   70],
       [ 435, 1523, 1356,  750]])

### Chi2 contingency independence test will conduct for array format observations to get a P-value,DOF & expected observations

In [9]:
chi2_contingency(Observations)

(1.595945538661058,
 0.6603094907091882,
 3L,
 array([[  42.76531299,  146.81287862,  131.11756787,   72.30424052],
        [ 442.23468701, 1518.18712138, 1355.88243213,  747.69575948]]))

In [11]:
#P_Value= 0.66 ,DOf= 3 

### p_Value(0.66030949) > aplha(0.05), We will accept null hypothesis. hence,
*** Male-female buyer rations are similar across regions.***